runtime/onert/core/include/ir/Index.h

```diff
@@ -47,10 +49,12 @@ using OriginIndex = ::onert::util::Index<uint32_t, OriginIndexTag>;
 template <typename IndexType>
 std::ostream &_index_print_impl(std::ostream &o, const std::string &prefix, IndexType index)
 {
+  std::ostringstream oss;
   if (index.undefined())
-    return o << prefix << std::string("?");
+    oss << prefix << std::string("?");
   else
-    return o << prefix << index.value();
+    oss << prefix << index.value();
+  return o << std::setw(3) << oss.str();
```

Comment:
(Optional)
How about use width=4 and `std::right`? Operand number can be three digit number in a model.

Reply:
@ragmani Thank you for suggestion. Here is the output based on your suggesion:

```
[LoweredTrainableGraph] dump before mandatory passes
[   GraphDumper  ] {
[   GraphDumper  ]     %4 =   @0_FullyConnected(  %0,   %1,   %?)
[   GraphDumper  ]     %5 =   @1_FullyConnected(  %4,   %2,   %?)
...
[  insertPermute ] Permute Op inserted, node index :   @4
[  insertPermute ]   - Input (original) Operand :   %6(train)
[  insertPermute ]   - Output(inserted) Operand :   %9(builtin)
...
[      Lower     ] Operand   %0 Info
[      Lower     ]   - Shape     : { 32 784 }
[      Lower     ]   - Def/Uses  : Def {   @? } Uses {   @5 }
[      Lower     ]   - Data      : N/A
[      Lower     ]   - LowerInfo : Def { builtin(NHWC) } Uses { builtin(UNKNOWN) }
...
[LoweredTrainableGraph] Graph Input :   %0
[LoweredTrainableGraph] Graph Input :   %7
[LoweredTrainableGraph] Graph Output :   %9
[   GraphDumper  ] {
[   GraphDumper  ]    %10 =   @5_Permute(  %0)
[   GraphDumper  ]     %4 =   @0_FullyConnected( %10,   %1,   %?)
[   GraphDumper  ]     %5 =   @1_FullyConnected(  %4,   %2,   %?)
...
[StaticShapeInferer]     %6, Static, shape : {32 10}
[StaticShapeInferer]     %5, Static, shape : {32 64}
[StaticShapeInferer]     %4, Static, shape : {32 64}
[StaticShapeInferer]     %3, Static, shape : {10 64}
[StaticShapeInferer]     %2, Static, shape : {64 64}
[StaticShapeInferer]     %1, Static, shape : {64 784}
[StaticShapeInferer]     %0, Static, shape : {32 784}
[StaticShapeInferer]     %7, Static, shape : {32 10}
[StaticShapeInferer]     %8, Static, shape : {32}
...
[   BP_PLANNER   ] CLAIM( %11): 0, 1280
[   BP_PLANNER   ] CLAIM( %10): 1280, 100352
[   BP_PLANNER   ] CLAIM(  %6): 101632, 1280
[   BP_PLANNER   ] CLAIM(  %5): 102912, 8192
[   BP_PLANNER   ] CLAIM(  %4): 111104, 8192
...
[  TensorManager ]           TENSOR   %8 : 0x565018f14ac0
[  TensorManager ]           TENSOR   %4 : 0x565018f12ac0
```

Right alignment looks better.

But you may notice there are additional 1 space for operand index.

- Width=3
```
[    Linearize   ]  %4 =  @0_FullyConnected(%10,  %1,  %?)
```
- Width=4
```
[    Linearize   ]   %4 =   @0_FullyConnected( %10,   %1,   %?)
```

Also, for 3 digits number, the output still shows the full number though it is not aligned.

I am okay for your suggestion (width=4 + right alignment) or (width=3 + right alignment).

Reply:
> - Width=4
```
[    Linearize   ]   %4 =   @0_FullyConnected( %10,   %1,   %?)
```

How about remove a white space from ", " in `formatOperandIndexSequence`?

Reply:
@ragmani I am okay for your suggestion. Let's get other's opinion also.